packages/docs/overview.md

```diff
@@ -84,29 +84,31 @@ The Governance contract is set as “owner” for all of the Celo Core Contracts
+Celo enables a family of stable-value ERC-20 tokens whose values can track the value of any asset, including fiat currencies, commodities, and even natural resources.  The first stablecoin that will be supported is the Celo Dollar, which tracks the value of the U.S. Dollar.  Celo Gold serves as the collateral for Celo Dollar, which is redeemable for 1 US Dollar worth of Celo Gold, ensuring that transactions can occur quickly, cheaply and reliably on-chain.
 
-{% hint style="success" %}
-**Roadmap**: Celo envisages a number of stable currencies pegged to different fiat currencies. The Celo Dollar will be pegged to the US Dollar on the future Celo production network.
-{% endhint %}
+Celo's stability mechanism allows users to create a new Celo Dollar by sending 1 US Dollar worth of Celo Gold to the reserve, or burn a Celo Dollar by redeeming it for 1 US Dollar worth of Celo Gold.
+
+This mechanism relies on a series of Oracles, or information feeds from exchanges external to the network, to report the Celo Gold to US Dollar market rate. To minimize the risk of a run on Celo Gold collateral when these reported values are inaccurate or out-of-date, Celo uses an on-chain constant-product-market-maker model, inspired by the [Uniswap](https://uniswap.io/) system. This mechanism adjusts the redemption price of Celo Gold until either arbitrage occurs (so that the on-chain price dynamically adjusts until the offered rate meets the external rate) or Oracles reset the on-chain price.
+
+The Celo protocol ensures that there is sufficient Celo Gold collateral to redeem the amount of Celo Gold in circulation through several sources. These include a [stability fee](celo-codebase/protocol/stability/stability-fees.md) levied on Celo Dollar balances, a transfer from [epoch rewards](celo-codebase/protocol/proof-of-stake/community-fund.md#bolster-reserve), plus the proceeds from the spread when interacting with the on-chain market-maker mechanism.  
 
-Celo's stability mechanism can be seen as the decentralized version of the one-to-one issuance and redemption mechanisms of most fiat-backed stablecoins. Users create new Celo Dollar by sending 1 US Dollar worth of Celo Gold to the reserve, or burn a Celo Dollar by redeeming them for 1 US Dollar worth of Celo Gold.
+In addition, a back-up reserve of cryptocurrencies is held off-chain.  This off-chain reserve is managed to preserve value and minimize volatility by maintaining a diversified portfolio of cryptocurrencies through algorithmic rebalancing trading and periodically "topping-up" the Celo Gold collateral available to ensure it exceeds the amount required to redeem Celo Dollars in circulation.  The approved cryptocurrencies, distribution ratios, and rebalancing period are all subject to on-chain governance.
```

Comment:
Nit: Looks like two sentences in this line have an extra leading space. "This off..." and "The approved..."
Is that a markdown feature?

Reply:
copy-paste :-/